app/components/appeals-app-services/variables.tf

```diff
@@ -338,11 +338,6 @@ variable "service_bus_appeals_bo_document_topic_id" {
   type        = string
 }
 
-variable "service_bus_listed_building_topic_id" {

```

Comment:
it might not be planned but we are using it to populate the FO db with listed building records so assuming we will need to come up with another means of populating that db table?

Reply:
Correct, it is on my list to decide our approach for that before ODW authors that data.